scripts/crawl.py

```diff
@@ -44,6 +40,20 @@ def parse_args():
     with open(config_path, "r") as f:
         config = json.load(f)
 
+    if 'conp-dataset_path' not in config.keys():
+        raise Exception(
+            '"conp-dataset_path" not configured in ' + config_path + ','
+            'please configure it as follows: \n'
+            '  "conp-dataset_path": "PATH TO conp-dataset DIRECTORY",'
+        )
+
+    # import the crawler packages
+    sys.path.append(config['conp-dataset_path'])
+    global ZenodoCrawler
+    global OSFCrawler
```

Comment:
Do we really need to add global variables? It's not great practice in general.

Reply:
@glatard I was not too keen either about the global variables. I slept on it and found another way to import the classes directly in the __main__ function. Let me know what you think.